sketch/portfolio-page.php

```diff
@@ -11,36 +11,48 @@
 		<main id="main" class="site-main" role="main">
 
 		<?php if ( sketch_has_featured_posts( 1 ) ) : ?>
-		    <div class="featured-content">
-		        <?php get_template_part( 'content-featured' ); ?>
-		    </div>
+			<div class="featured-content">
+				<?php get_template_part( 'content-featured' ); ?>
+			</div>
 		<?php endif; ?>
 
-		<?php if ( ! get_theme_mod( 'sketch_hide_portfolio_page_content' ) ) : ?>
-			<?php while ( have_posts() ) : the_post(); ?>
+		<?php if ( post_password_required() ) : ?>
 
-				<?php the_title( '<header class="page-header"><h1 class="page-title">', '</h1></header>' ); ?>
+			<?php the_title( '<header class="page-header"><h1 class="page-title">', '</h1></header>' ); ?>
+			<?php the_content(); ?>
```

Comment:
We should wrap this in `.page-content` so that it matches how the revealed content is rendered.

```suggestion
			<div class="page-content">
				<?php the_content(); ?>
			</div>
```

Reply:
I've wrapped it with a similar container as used below.